src/pj_obs_api.c

```diff
@@ -296,10 +315,44 @@ size_t proj_transform (
         *z = coord.xyzt.z;
     if (nt==1)
         *t = coord.xyzt.t;
-    
+
     return i;
 }
 
+/*****************************************************************************/
+int proj_transform_obs (PJ *P, enum proj_direction direction, size_t n, PJ_OBS *obs) {
+/******************************************************************************
+    Batch transform an array of PJ_OBS.
+
+    Returns 0 if all observations are transformed without error, otherwise
+    returns error number.
```

Comment:
Thinking a bit more about this... At the moment the `proj_transform_*` functions are very crude. I just threw them together quickly to fill out the blanks in the last API PR by @busstoptaktik. Error handling needs to be improved. I can think of two different solutions:

1. Mimic `pj_transform`. That's more or less what the functions do now, except they don't distinguish between transient and permanent errors. It should be easy to implement the transient/permanent bit.

2. Return the number of succesfully transformed coordinates and let the user ask for the error number by calling `proj_errno`. In case different errors are returned during the loop the user will get the last returned error-number. Potentially that is a problem.

In both cases `HUGE_VAL`'s are returned for coordinates that fail to transform. Thoughts?